package.json

```diff
@@ -33,17 +33,18 @@
     "eslint": "^2.11.1",
     "eslint-config-brigade": "^2.0.3",
     "eslint-plugin-react": "^5.1.1",
+    "in-publish": "^2.0.0",
     "jasmine-core": "^2.1.3",
     "karma": "^0.13.9",
     "karma-chrome-launcher": "^0.2.0",
     "karma-cli": "^0.1.0",
     "karma-firefox-launcher": "^0.1.6",
     "karma-jasmine": "^0.3.6",
     "karma-webpack": "^1.7.0",
+    "mkdirp": "^0.5.1",
     "react": "^15.0.0",
     "react-dom": "^15.0.0",
     "rimraf": "^2.5.2",
-    "mkdirp": "^0.5.1",
```

Comment:
This was moved because this file was manually edited at some point. Using the CLI for installing deps keeps them in alphabetical order.